source/screens/Splash/Splash.tsx

```diff
@@ -2,12 +2,33 @@ import * as React from 'react';
 import {Loader} from 'semantic-ui-react';
 
 import './Splash.scss';
+import {Link, useNavigate} from 'react-router-dom';
 import Screen from '../Screen';
+import Button from '../../components/Buttons';
 
-const Splash: React.FC = () => {
+type Props = {
+  newAccount: boolean;
+  onCreate: () => void;
+};
+const Splash: React.FC<Props> = ({newAccount, onCreate}) => {
+  const navigate = useNavigate();
+  const handleCreateMnemonic = () => {
+    onCreate();
+    navigate('/create');
+  };
   return (
     <Screen className="splash-screen">
-      <Loader className="splash-screen__loader" active inverted />
+      {!newAccount && (
+        <Loader className="splash-screen__loader" active inverted />
+      )}
+      {newAccount && (
```

Comment:
I suggest to move this logic into a dedicated component (Call it Intro/Startup or something like that). Let's just leave Splash screen as is - loader and nothing more.